contracts/mocks/oracle/IthacaFeed.sol

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.6.12;
+pragma experimental ABIEncoderV2;
+
+import {IIthacaFeed} from "../../protocol/ithaca/IIthacaFeed.sol";
+
+contract IthacaFeed is IIthacaFeed {
+  ClientParams clientData;
+
+  function setData(ClientParams memory param, uint64 backendId) external {
+    clientData = param;
+  }
+
+  function getClientData(
+    address client
+  ) public view override returns (address, int256, int256, uint256, uint256) {
+    return (
+      client,
+      clientData.maintenanceMargin,
+      clientData.mtm,
+      clientData.collateral,
+      clientData.vaR
+    );
+  }
+}
```

Comment:
```
// SPDX-License-Identifier: UNLICENSED
pragma solidity 0.6.12;
pragma experimental ABIEncoderV2;

import {IIthacaFeed} from '../../protocol/ithaca/IIthacaFeed.sol';
import '@openzeppelin/contracts/access/Ownable.sol';

contract IthacaFeed is IIthacaFeed, Ownable {
  ClientParams clientData;

  function setData(ClientParams memory param, uint64 backendId) external onlyOwner {
    clientData = param;
    emit ClientUpdated(backendId);
  }

  function getClientData(
    address client
  ) public view override returns (address, int256, int256, uint256, uint256) {
    return (
      client,
      clientData.maintenanceMargin,
      clientData.mtm,
      clientData.collateral,
      clientData.vaR
    );
  }
}
```

Reply:
This is a mock contract, renamed it to MockIthacaFeed.sol